### DISCUSSION TITLE
Getting random ETX

### DISCUSSION THREAD

**Discussion by i-zanis** [JetBrains/ideavim#523] [Q&A]
Hello Alex & Team,

Thank you for all your contributions. I have been using the plugin for more than 2 years. For a few months now, I am getting random ETX on my editor usually when I press Ctrl C (which I have mapped it to Escape). 

I don't know exactly what is the sequence, I cannot replicate it effectively but it has happened to 3 computers that I use Intellij Idea Ultimate.
![image](https://user-images.githubusercontent.com/55964867/175948446-d0ef9541-703d-4aca-8655-03bba3ca4b8e.png)

**Comment by AlexPl292** [comment]
Hi! Thank you for your kind words.
Maybe this character is included in some of your mapping (accidentally)? Can you please attach your `.ideavimrc` file?

**Reply by i-zanis** [reply]
@AlexPl292 
It happens sometimes on Ctrl + C

"" Source your .vimrc
"source ~/.vimrc

let mapleader = " " " Make space the Leader key

" https://github.com/JetBrains/ideavim/blob/master/doc/ideajoin-examples.md
set ideajoin       " use IntelliJ's intelligent way of joining lines
set showmode       " show vim mode at the bottom left
set nu             " show numbers in the gutter
" set relativenumber " show relative numbers 1-9 in the gutter

" Show a few lines of context around the cursor. Note that this makes the
" text scroll if you mouse-click near the start or end of the window.
set scrolloff=3
set visualbell
set incsearch " Do incremental searching.

" Make tab navigation easier
noremap E gT
noremap R gt

set clipboard=unnamedplus
set hls
set highlightedyank " highlights the yank area
let g:highlightedyank_highlight_duration = "1000"

" Don't use Ex mode, use Q for formatting.
map Q gq

" add navigation because of IntelliJ - IdeaVim keybindings conflicts
nmap g] :action GotoImplementation<CR>
nmap g[ :action GotoSuperMethod<CR>
noremap ghc <ESC>:action CallHierarchy<CR>
noremap ght <ESC>:action TypeHierarchy<CR>
noremap ghm <ESC>:action MethodHierarchy<CR>

" built-in navigation to navigated items works better than the Jumplist
nnoremap <c-o> :action Back<cr>
nnoremap <c-i> :action Forward<cr>
" but preserve ideavim defaults
nnoremap <C-S-o> <c-o>
nnoremap <C-S-i> <c-i>

" Ctrl F keybinding conflict with page down and built in search looks better
nnoremap / :action Find<cr>
" but preserve ideavim search
nnoremap g/ /
